packages/web-app-files/src/components/SideBar/Shares/Collaborators/InviteCollaborator/RecipientContainer.vue

```diff
@@ -59,7 +60,7 @@ export default {
   },
 
   async created() {
-    if (this.capabilities.files_sharing.user.profile_picture && this.hasAvatar) {
+    if (this.capabilities.files_sharing.user.profile_picture && (this as any).hasAvatar) {
```

Comment:
Should probably be `this.formattedRecipient.hasAvatar` but then the unit test fails ... needs investigation